db/migrations/20221108235926-create-pushdevices.js.js

```
@@ -0,0 +1,50 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up (queryInterface, Sequelize) {
+    await queryInterface.createTable('push_devices', {
+      device_id: {
+        type: Sequelize.STRING(256),
+        allowNull: false,
+        primaryKey: true,
+      },
+      push_provider: {
+        type: Sequelize.ENUM(['ios', 'android']),
+        allowNull: false,
+        defaultValue: 'android',
```

I don't think we should have a default value here. If some code tries to register a device without specifying the provider, I think it's better to throw an error than to suppose a value.

What do you think about this?